README.md

```diff
@@ -0,0 +1,59 @@
+# Math Magicians
```

Comment:
- [ ] **Optional:** Your document has a wrong title. I apologize that this wasn't pointed out earlier. I also strongly recommend making changes to this effect.